### DISCUSSION TITLE
Installation and Configuration - Fedora Messagging Module

### DISCUSSION THREAD

**Discussion by pilomat** [WeblateOrg/weblate#9455] [Q&A]
Hi Everyone, 

I apologize for the junior question, but I new in this project.

We are struggling to understand to integrate this module and we would like to ask a step by step guide to install this module.

As far as we understood this module has been already installed in the weblate docker image. Can we configure it throught the docker compose configuration ? 

Therefore, How can I access to the celery queue in order to subscribe the messages ? 

Thanks in advance

**Comment by nijel** [comment]
Asked and answered in https://github.com/WeblateOrg/fedora_messaging/issues/246#issuecomment-1609456359
